plugins/cost-insights/api-report.md

```diff
@@ -178,10 +178,10 @@ export type BarChartProps = {
   resources: ResourceData[];
   responsive?: boolean;
   displayAmount?: number;
-  options?: Partial<BarChartData>;
-  tooltip?: ContentRenderer<TooltipProps>;
-  onClick?: RechartsFunction;
-  onMouseMove?: RechartsFunction;
+  options?: Partial<BarChartOptions>;
+  tooltip?: ContentType<ValueType, NameType>;
+  onClick?: (...args: any[]) => void;
+  onMouseMove?: (...args: any[]) => void;
```

Comment:
Was there no more precise definition for these event handlers? Something that has the event object declared or so. If we promise absolutely _nothing_ for these parameters like this, then the type could just as well have been `() => void`

Reply:
This was the definition of the `RechartsFunction` type on version v1.8.5:

```ts
export type RechartsFunction = (...args: any[]) => void;
```

Since the type was removed on v2.0.0, I just replaced it with its equivalence.